src/main/java/org/minions/devfund/alejandro/MainWhackAMole.java

```diff
@@ -0,0 +1,54 @@
+package org.minions.devfund.alejandro;
+import java.util.Scanner;
+
+/**
+ * class for main wack a mole game.
+ */
+public abstract class  MainWhackAMole {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "MainWhackAMole" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)